doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -676,26 +681,14 @@ fantray0    N/A  fantray0.fan      55%       intake     Present        OK  20230
 fantray1    N/A  fantray1.fan      56%       intake     Present        OK  20230728 06:41:17
 ```
 
-#### 3.4.1 Reboot Cause
+#### 3.4.1 Reboot Cause CLIs
 * There are two CLIs "show reboot-cause" and "show reboot-cause history" which are applicable to both DPUs and the Switch. However, when executed on the Switch the CLIs provide a consolidated view of reboot cause as shown below.
-* Each DPU will update its reboot cause history in the Switch ChasissStateDB upon boot up. The recent reboot-cause can be derived from that list of reboot-causes.
-* The switch side PMON will copy this into the stateDB so that the existing workflow will not be affected.
-* The get_reboot_cause API will return the current reboot-cause of the module.
-
-#### REBOOT_CAUSE DB schema
-```
-Key: "REBOOT_CAUSE|2023_06_18_14_56_12"
+* Each DPU will update its reboot cause history in the Switch ChassisStateDB upon boot up.
+* The PMON on the DPU side will be responsible to update the switch side chassisStateDB on DPU boot up, using the push model specified in [section: 3.2.4 of SONiC Chassis Platform Management & Monitoring HLD](https://github.com/sonic-net/SONiC/blob/master/doc/pmon/pmon-chassis-design.md)
+* Though how DPU pmon updates this is vendor dependent, it is recommended to use the sonic telemetry agent to align with the existing SONiC implementation.
+* The DPUs will limit the number of history entries to a maximum of ten.
```

Comment:
@rameshraghupathy Why DPU pmon updates needs to be vendor dependent?

Reply:
There is no guarantee that the SONiC running on the DPUs will necessarily be running Telemetry.